internal/eval/eval.go

```diff
@@ -4627,3 +4627,39 @@ func evalJSONSTRAPPEND(args []string, store *dstore.Store) []byte {
 	obj.Value = jsonData
 	return clientio.Encode(resultsArray, false)
 }
+
+// evalRANDOMKEY returns a random key from the currently selected database.
+func evalRANDOMKEY(args []string, store *dstore.Store) *EvalResponse {
+	if len(args) > 0 {
+		return &EvalResponse{Result: nil, Error: errors.New(string(diceerrors.NewErrArity("RANDOMKEY")))}
+	}
+
+	availKeys, err := store.Keys("*")
+	if err != nil {
+		return &EvalResponse{Result: nil, Error: diceerrors.ErrUnableToFetchKeys}
```

Comment:
`ErrGeneral` would be more suitable since `UnableToFetchKeys` is not an general error I feel

Reply:
ErrGeneral does not really tell what the error is and might end up getting used somewhere else despite it not meaning the same thing.

Reply:
`ErrGeneral` basically takes the string and adds ERR to the string. Like the `ErrUnableToFetchKeys` you have defined. I think `ErrUnableToFetchKeys` might not be used anywhere else therefore using ErrGeneral would be better imo. What do you think?

```go
ErrGeneral = func(err string) error {
		return fmt.Errorf("ERR %s", err) // General error format for various commands.
	}
```